submarine-client/src/main/java/org/apache/submarine/client/cli/AbstractCli.java

```diff
@@ -42,11 +42,11 @@ public abstract int run(String[] args)
 
   @Override
   public void setConf(Configuration conf) {
-    clientContext.setSubmarineConfig(conf);
+    clientContext.setConfiguration(conf);
   }
 
   @Override
   public Configuration getConf() {
-    return clientContext.getSubmarineConfig();
+    return clientContext.getYarnConfig();
```

Comment:
`setConfiguration()` and `getYarnConfig()`, Why are these two functions inconsistent?